### ISSUE TITLE
Update sucker_punch gem

### ISSUE THREAD

**Issue by repeatedly** [fluent/fluentd-ui#204]
Latest sucker_punch replaces celluloid with concurrent-ruby.
It is good for reducing gem dependencies.

**Comment by okkez** [comment]
sucker_punch gem is updated in #222 (1.6.0) and #223 (2.0.4)

```
$ gem dep sucker_punch
Gem sucker_punch-1.6.0
  celluloid (~> 0.17.2)
  pry (>= 0, development)
  rake (>= 0, development)
  rspec (>= 0, development)

Gem sucker_punch-2.0.4
  concurrent-ruby (~> 1.0.0)
  minitest (>= 0, development)
  pry (>= 0, development)
  rake (~> 10.0, development)
```

**Comment by okkez** [comment]
Released v0.4.5.
